lib/uktest/README.md

```diff
@@ -0,0 +1,131 @@
+# uktest: Unikraft’s Testing Framework
+
+`uktest` is Unikraft's internal testing framework.
+It provides a flexible API for performing unit testing internal libraries, methods and more in order to safeguard changes which affect existing functionality.
```

Comment:
Apologies, I see we're missing an `of` in the sentence:

```diff
- It provides a flexible API for performing unit testing internal libraries, methods and more in order to safeguard changes which affect existing functionality.
+ It provides a flexible API for performing unit testing of internal libraries, methods and more in order to safeguard changes which affect existing functionality.
```

Reply:
Ah, I missed that.